UnboundLib/Utils/CardManager.cs

```diff
@@ -122,78 +119,92 @@ public static CardInfo GetCardInfoWithName(string cardName)
             return cards.ContainsKey(cardName) ? cards[cardName].cardInfo : null;
         }
 
-        public static void EnableCards(CardInfo[] cards, bool saved = true)
+        public static void EnableCards(CardInfo[] cardInfos, bool saved = true)
         {
-            foreach (var card in cards)
+            foreach (var card in cardInfos)
             {
                 EnableCard(card, saved);
             }
         }
 
-        public static void EnableCard(CardInfo card, bool saved = true)
+        public static void EnableCard(CardInfo cardInfo, bool saved = true)
         {
-            if (!activeCards.Contains(card))
+            if (!activeCards.Contains(cardInfo))
             {
-                activeCards.Add(card);
-                activeCards = new ObservableCollection<CardInfo>(activeCards.OrderBy(i => i.cardName));
+                activeCards.Add(cardInfo);
+                activeCards = new ObservableCollection<CardInfo>(activeCards.OrderBy(i => i.gameObject.name));
                 activeCards.CollectionChanged += CardsChanged;
             }
-            if (inactiveCards.Contains(card))
+            if (inactiveCards.Contains(cardInfo))
             {
-                inactiveCards.Remove(card);
+                inactiveCards.Remove(cardInfo);
             }
 
-            cards[card.cardName].enabled = true;
+            string cardName = cardInfo.gameObject.name;
+            if (!cardName[0].Equals('_'))
+            {
+                cardName = cardInfo.cardName;
+            }
+            if (!cards.ContainsKey(cardName)) return;
```

Comment:
Vanilla card prefabs do not have the naming scheme on the objects like the modded cards, this just recognizes which one is which

Reply:
Relying on creators to not use `_` as the first character in their card name seems fault-prone and unnecessary. Instead why not just check the card's categories to see if it's a default card? Or better yet (since it wouldn't require adding a default category to every vanilla card) just check to see if it has a `CustomCard` component.